### ISSUE TITLE
RLOO的normalization似乎错误了？

### ISSUE THREAD

**Issue by millioniron** [OpenRLHF/OpenRLHF#843]
如题，

        if args.advantage_estimator == "rloo":
            rewards = torch.cat([experience.info["reward"] for experience in experiences])
            rewards = rewards.reshape(-1, args.n_samples_per_prompt).to(device="cuda")
            baseline = (rewards.sum(-1, keepdim=True) - rewards) / (args.n_samples_per_prompt - 1)
            rewards = rewards - baseline
            rewards = rewards.flatten().to(device="cpu").chunk(len(experiences))

RLOO论文似乎是将不同的样本进行normalize，代码似乎和grpo一样，都是在同一样本的不同采样上？

**Comment by millioniron** [comment]
解决，在后面replay_buffer的部分
